source/compiler/sc2.c

```diff
@@ -2141,8 +2141,8 @@ char *sc_tokens[] = {
          "assert", "*begin", "break", "case", "char", "const", "continue", "default",
          "defined", "do", "else", "emit", "__emit", "*end", "enum", "exit", "for",
          "forward", "goto", "if", "native", "new", "operator", "public", "return",
-         "sizeof", "sleep", "state", "static", "stock", "switch", "tagof", "*then",
-         "while",
+         "sizeof", "sleep", "state", "__static", "__static_assert", "static_check", "stock",
```

Comment:
What is `__static`?

Reply:
It looks like the `__` was added on the wrong symbol, it should be:

```c
"static", "__static_assert", "__static_check"
```

Reply:
That was a typo while adding the underscore prefixes.